dotCMS/src/main/java/com/dotmarketing/business/query/QueryUtil.java

```diff
@@ -235,5 +235,19 @@ private static void buildComplexCriteria(Map<String, String> dbColToObjectAttrib
 
 		return filteredResults; 
 	}
-	
+
+	/**
+	 *
+	 * @param luceneQuery
+	 * @return
+	 */
+	public static String removeQueryPrefix(final String luceneQuery){
+		final String cleanedUpQuery;
+		if(luceneQuery.startsWith("query_")){
+			cleanedUpQuery = luceneQuery.replace( "query_", "" );
```

Comment:
replace "" by StringPool.BLANK

Reply:
you could replace by (luceneQuery.startsWith("query_"))?luceneQuery.replace( "query_", StringPool.BLANK ): luceneQuery;